cmd/puppetdb/pdb.go

```diff
@@ -90,30 +91,79 @@ func execute(api string, query string) {
 	cli.PrintString(data)
 }
 
-func parseInput(command string) (string, string) {
-	blocks := strings.Split(command, " ")
-	var api, query string
+func extractString(str, start, end string) (result string) {
+	s := strings.Index(str, start)
+	s--
+	if s == -1 {
+		return
+	}
+	s += len(start)
+	e := strings.LastIndex(str, end)
+	e++
+	if e == -1 {
+		return
+	}
+	return str[s:e]
+}
+
+func parseInput(command string) (string, string, puppetdb.Pagination) {
```

Comment:
Could you move the parseInput and extractString functions into internal/cli/helpers ?   I think they are getting a bit long to keep in the main